e3nn/util/dataset/molecules.py

```diff
@@ -149,3 +149,135 @@ def download(self):
 
         self.files = sorted(glob.glob(os.path.join(xyz_path, "*.xyz")))
         assert len(self.files) == 133885
+
+
+
+import torch
+
+import os
+from functools import partial
+from itertools import zip_longest
+
+from pymatgen.core.structure import Molecule
+from pymatgen.core.periodic_table import Element
+from torch_geometric.data import Dataset
+
+
+def to_one_hot(ind_arr, n_classes):
+    one_hot = torch.zeros((len(ind_arr), n_classes))
+    one_hot[torch.arange(len(ind_arr)), ind_arr] = 1.
+    return one_hot
+
+
+# function that constructs graph given 
+def molecule_from_file_to_graph_sphere_connectivity_5_no_self_interactions(molecule_file, y):
+    return molecule_from_file_to_graph_base(molecule_file, y, connectivity_func=connections_within_sphere_5_no_self_interactions)
+
+def connections_within_sphere_5_no_self_interactions(mol):
```

Comment:
I'm not sure this function is necessary as it is basically a lambda version of `connection_within_sphere` with r_cut == 5. Is there a specific reason why you would like to have this in the codebase?